pkg/vehicle/media.go

```diff
@@ -0,0 +1,58 @@
+package vehicle
+
+import (
+	"context"
+
+	"github.com/teslamotors/vehicle-command/pkg/protocol/protobuf/carserver"
+)
+
+func (v *Vehicle) MediaNextTrack(ctx context.Context) error {
+	return v.executeCarServerAction(ctx,
+		&carserver.Action_VehicleAction{
+			VehicleAction: &carserver.VehicleAction{
+				VehicleActionMsg: &carserver.VehicleAction_MediaNextTrack{},
+			},
+		})
+}
+
+func (v *Vehicle) MediaPreviousTrack(ctx context.Context) error {
+	return v.executeCarServerAction(ctx,
+		&carserver.Action_VehicleAction{
+			VehicleAction: &carserver.VehicleAction{
+				VehicleActionMsg: &carserver.VehicleAction_MediaPreviousTrack{},
+			},
+		})
+}
+
+func (v *Vehicle) MediaNextFavorite(ctx context.Context) error {
+	return v.executeCarServerAction(ctx,
+		&carserver.Action_VehicleAction{
+			VehicleAction: &carserver.VehicleAction{
+				VehicleActionMsg: &carserver.VehicleAction_MediaNextFavorite{},
+			},
+		})
+}
+
+func (v *Vehicle) MediaPreviousFavorite(ctx context.Context) error {
+	return v.executeCarServerAction(ctx,
+		&carserver.Action_VehicleAction{
+			VehicleAction: &carserver.VehicleAction{
+				VehicleActionMsg: &carserver.VehicleAction_MediaPreviousFavorite{},
+			},
+		})
+}
+
+func (v *Vehicle) MediaVolumeRelative(ctx context.Context, delta int32) error {
```

Comment:
It looks like the car just looks at the sign of delta. So +10 and +1 do the same thing (increase volume by one unit) and -10 and -1 do the same thing (decrease volume by one unit). So we should probably split this into MediaVolumeUp and MediaVolumeDown, which correspond directly with the REST API.